config/manifests/metallb-frr-k8s-prometheus.yaml

```diff
@@ -3279,6 +3279,17 @@ spec:
     tlsConfig:
       insecureSkipVerify: true
   - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    metricRelabelings:
```

Comment:
Can we have these also set by default in the helm chart? Or the idea is to document the change if a user wants to keep backward compatibility?

Reply:
If so, we should remember of doing so when we document it.

Reply:
yep my idea was to document it as it depends on the pod/service monitors setup

Reply:
then let's file an issue to document, with next release as target